src/course-outline/hooks.jsx

```diff
@@ -33,6 +35,8 @@ const useCourseOutline = ({ courseId }) => {
   const [isDisabledReindexButton, setDisableReindexButton] = useState(false);
   const [showSuccessAlert, setShowSuccessAlert] = useState(false);
   const [showErrorAlert, setShowErrorAlert] = useState(false);
+  const [isHighlightsModalOpen, openHighlightsModal, closeHighlightsModal] = useToggle(false);
+  const [currentSection, setCurrentSection] = useState({});
```

Comment:
i think `currentSection` would be better to store it in slice

Reply:
Done